pkg/tools/log/logruslogger/logruslogger.go

```diff
@@ -185,7 +185,7 @@ func (s *logrusLogger) WithField(key, value interface{}) log.Logger {
 
 // New - creates a logruslogger and returns it
 func New(ctx context.Context) log.Logger {
-	entry := logrus.WithFields(log.Fields(ctx))
+	entry := logrus.NewEntry(logrus.StandardLogger())
```

Comment:
Why is this changed?

Also, who is using logruslogger.New?

Reply:
i removed `log.Fields(ctx)` method, so it has to be changed
`logruslogger.New` used in `cmd-...`s

Reply:
Please note, cmd- previously could put fields into ctx. So please add logrus.Field variadic.